src/utils.js

```diff
@@ -0,0 +1,50 @@
+export const SYSTEM_PROMPT = `
+You are an expert software engineer reviewing a pull request on Github. Lines that start with "+" have been added, lines that start with "-" have been deleted. Use markdown for formatting your review.
+
+Desired format:
+### Description
+<description_of_PR> // How does this PR change the codebase? What is the motivation for this change?
+
+### Changes
+<list_of_changes> // Describe the main changes in the PR, organizing them by filename
+
+### Security Hotspots
+<list_of_security_hotspots> // Describe locations for possible vulnerabilities in the change, ordered by risk. Do not include a vulnerability unless it is likely to present a real security risk. If there are none, omit this section.
+
+### Other Issues
+<list_of_issues> // Describe any other major issues with this code such as license incompatibilities introduced by a dependency. If there are none, omit this section.
+\n`
+
+export async function explainPatchHelper (patchBody, owner, repo, models, debug, getResponse) {
+  models = Array.isArray(models) ? models : models.split(' ')
+
+  let model = null
+  let response = null
+
+  const userPrompt = `Repository: https://github.com/${owner}/${repo}\n\nThis is the PR diff\n\`\`\`\n${patchBody}\n\`\`\``
+
+  if (debug) {
+    console.log(`user_prompt:\n\n${userPrompt}`)
+  }
+
+  for (let i = 0; i < models.length; i++) {
+    try {
+      model = models[i]
+      response = await getResponse(userPrompt, model)
+      break
+    } catch (e) {
+      if (i + 1 === models.length) {
+        // last model
+        throw e
+      }
+
+      console.log(e)
+      continue
+    }
+  }
+
+  response = response.replaceAll('### Changes', '<details>\n<summary><i>Changes</i></summary>\n\n### Changes')
+  response = response.replaceAll('### Security Hotspots', '</details>\n\n### Security Hotspots')
```

Comment:
Since `Security Hotspots` section can now be omitted, this should be adapted, to get the first `###` after the `### Changes` section, otherwise it breaks the formatter.

This is custom github syntax, and the AI are not aware of